_posts/2023-11-20-sct-course.md

```diff
@@ -55,4 +57,4 @@ To come.
 
 ## Organizers
 
-To come.
+[SCT team](https://github.com/spinalcordtoolbox/spinalcordtoolbox/people).
```

Comment:
https://github.com/spinalcordtoolbox/spinalcordtoolbox/people returns a 404 error

Reply:
I can see it just fine, so I think it may be a permissions issue? @jcohenadad and I have "Owner" status in the `spinalcordtoolbox` organization, while your account has "Member" status.

Good catch, though! I'll see if I can find an alternate link that has more visibility.

Reply:
The org list of people seems to be visible when I open it in a private tab? (Though, it will only show people who have made their org membership publicly visible.)

https://github.com/orgs/spinalcordtoolbox/people

I do notice that this list contains people who have since left the lab, so we may want to take some time to prune organization membership.

Reply:
great catch folks! yes, let's use the org

and yes, can someone pls take care of the pruning? thx!